files/docker/node/addons/healthcheck.sh

```diff
@@ -68,29 +70,28 @@ check_cncli_db() {
 
 
 # Function to check if the tip is successfully being sent to Pooltool
-check_cncli_send_tip() {
-    # Timeout in seconds for capturing the log entry
-    log_entry_timeout=99
-
+check_cncli_sendtip() {
     # Get the process ID of cncli
     process_id=$(pgrep -of cncli) || {
         echo "Error: cncli process not found."
         return 1  # Return 1 if the process is not found
     }
 
     # Get the current tip from the node
-    FIRST_TIP=$($CCLI query tip --testnet-magic ${NWMAGIC} | jq .block)
+    first_tip=$($CCLI query tip --testnet-magic ${NWMAGIC} | jq .block)
     # Capture the next output from cncli that is related to Pooltool
-    pt_log_entry=$(timeout $log_entry_timeout cat /proc/$process_id/fd/1 | grep --line-buffered "Pooltool" | head -n 1)
+    pt_log_entry=$(timeout $CNCLI_SENDTIP_LOG_TIMEOUT cat /proc/$process_id/fd/1 | grep --line-buffered "Pooltool" | head -n 1)
     # Get the current tip again
-    SECOND_TIP=$($CCLI query tip --testnet-magic ${NWMAGIC} | jq .block)
+    second_tip=$($CCLI query tip --testnet-magic ${NWMAGIC} | jq .block)
+    # If no output was captured...
     if [ -z "$pt_log_entry" ]; then
-        if [[ "$FIRST_TIP" -eq "$SECOND_TIP" ]]; then
-            echo "Unable to capture cncli output within $log_entry_timeout seconds, but node has not moved tip. (Current tip = $SECOND_TIP)."
-            return 0
+        tip_difference=$((second_tip - first_tip))
+        if [[ "$tip_difference" -le "$CNCLI_SENDTIP_ALLOWED_DRIFT" ]]; then
```

Comment:
Let's use `check_tip()` so we don't repeat code. Otherwise this looks good and ready to merge.

```suggestion
            if check_tip "${second_tip}" "${first_type}" "${CNCLI_SENDTIP_ALLOWED_DRIFT}" ; then
```

I'll rename DB_TIP in my branch after I rebase on your merge.